### DISCUSSION TITLE
AWS CloudWatch Log

### DISCUSSION THREAD

**Discussion by SleeperSmith** [Kong/kong#7383] [Ideas and feature requests]
A plugin to send logs to cloudwatch log?

**Comment by aditya232** [comment]
Has anyone started working on this? I'm considering this for a project and would like to controbute .

**Comment by coopr** [comment]
What advantages would this have over the existing logging plugins?

**Comment by BigBallard** [comment]
@coopr to get the logs into CloudWatch maybe? If an enterprise uses CW for log aggregation then that is advantage enough.
